client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -100,6 +100,11 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 				);
 				break;
 		}
+
+		recordOnboardingError( {
```

Comment:
This is being called as part of component render, which means the it keeps firing the event when the error message is shown. So:
1. Enter email
2. Enter a 1-character password
3. Submit
4. Start typing a new password to fix the issue, each time you type it sends another event.

Maybe you could argue that the error message should be cleared after the user starts typing, but in either case I don't think the event should be sent during the render. A re-render could be caused for all sorts of reasons.

Reply:
`handleSignUp()` is able to detect whether `createAccount` was successful, so maybe it can be added there?

Reply:
Can you tell I didn't test it properly :trollface: ?

Thanks for pointing this out!!